client/src/index.js

```diff
@@ -1,7 +1,14 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-
+import { BrowserRouter as Router, Route, Link } from "react-router-dom";
 import './index.css';
```

Comment:
only BrowserRouter as Router needs to be imported here. The others you can import where you use them.